doc/jobs/prerelease_jobs.rst

```diff
@@ -101,3 +112,32 @@ scripts which can be invoked manually in the same order:
   workspace.
   It also invokes the tool ``catkin_test_results --all`` to output a
   summary of all tests in the overlay workspace.
+
+Run the *prerelease* job on Travis
+----------------------------------
+
+Since it is easy to run a *prerelease* job locally it can also be run on Travis to either test every commit or pull request.
+The setup and invocation is the same as locally.
+The `.travis.yml <https://github.com/ros-infrastructure/ros_buildfarm/blob/master/.travis.yml>`_ file of the *ros_buildfarm* repository is a good starting point.
+
+Run for "custom" repositories
+-----------------------------
+
+Most job types require that the tested repository is being listed in a ROS distribution file.
+For the *prerelease* job that is not necessary.
```

Comment:
@130s I just tried to run a prerelease for the `baxter_app_rpc` repo. It fails to build though since one of the packages tries to find `actionlib` but doesn't declare that build dependency in the manifest: https://github.com/tork-a/baxter_app_rpc/blob/cbc9fbca6ac44dc6f2e0b105e5b42ad7b5a9589f/baxter_rpc_msgs/CMakeLists.txt#L5

I have created a dummy package in a branch of this repo (https://github.com/ros-infrastructure/ros_buildfarm/tree/dummy_package) which can be used to demonstrate the invocation and can also be used for testing the feature on Travis.

Please see #392 for future updates on this topic.